internal/cmd/login/login.go

```diff
@@ -107,7 +107,7 @@ func (c *loginCmd) run() error {
 
 	err = errors.Errorf(`No API access token provided. Please pass a valid token via stdin or run
 in interactive mode. You can generate a token here:
-%s/dashboard/settings/account/tokens?create.`+"\n", c.opts.Server)
+%s/dashboard/settings/account/tokens?create?origin=cli.`+"\n", c.opts.Server)
```

Comment:
This is not how multiple url params working... ? is separating the path from the params, & is used for separating params
```suggestion
%s/dashboard/settings/account/tokens?create&origin=cli.`+"\n", c.opts.Server)
```

Reply:
ouch, thank you! will fix